### PR TITLE
unsafe variance: reword note and fix the notes code

### LINKED ISSUE #444: unsafe variance ignore leaves spurious notes
```py
from typing import Generic

from basedtyping import T_co

class A(Generic[T_co]):
    def foo(self, t: T_co):  # type: ignore[unsafe-variance]
        pass
```
```
test.py: note: In member "foo" of class "A":
test.py:6:22: note: If this is intentional and you know what you are doing, you can ignore this line with 'unsafe-variance'
test.py:6:22: note: Error code "misc" not covered by "type: ignore" comment
Success: no issues found in 1 source file   
```

# workaround

Also ignore `misc`

### PR DESCRIPTION
**Pull request by KotlinIsland**
misc note was imposting as an attached note

- resolves #444

### CODE REVIEW

File: mypy/message_registry.py [hunk 1]
```diff
@@ -112,7 +112,10 @@ def with_additional_msg(self, info: str) -> ErrorMessage:
     "This usage of this covariant type variable is unsafe as an input parameter.",
     codes.UNSAFE_VARIANCE,
 )
-UNSAFE_VARIANCE_NOTE = "If this is intentional and you know what you are doing, you can ignore this line with 'unsafe-variance'"
+UNSAFE_VARIANCE_NOTE = ErrorMessage(
+    "If you are using the value in a 'variance safe' way, this error could be ignored",
```
Comment (DetachHead) [inline mypy/message_registry.py:116]: can this maybe link to some documentation or something explaining what "variance safe" actually means

Comment (KotlinIsland) [reply mypy/message_registry.py:116]: What does it means?

Comment (KotlinIsland) [reply mypy/message_registry.py:116]: I think something like "not storing the value". but that's a little confisung for conta outputs, which should say "don't return a stored value".

Comment (KotlinIsland) [reply mypy/message_registry.py]: - How does that look? I think when we #260 we can have a section there.

**Review by DetachHead — APPROVED** [review]

**Pull request merged**